### PR TITLE
fix: fix canon case study layout issue

### PR DESCRIPTION
**Pull request by antoniocapelo**
## Description - [Trello ticket](https://trello.com/c/GLfNfMth)

This MR aims to fix an issue found on our live site - on the canon case study.
It also removes some unnecessary paddings and simplifies the column numbers.

Note: There are still some small `px`s differences between this solution (and the one in production) and the designs regarding the width of our columns — but that's to be addressed in a `grid`-related ticked.

## Review template

The below is for reviewers of this PR to copy/paste into the review body and fulfill.

- Old grid break points: 0-599, 600-1095, 1095+
- New grid break points: 0-470, 471-552, 553-700, 701-899, 900-1196, 1197+

I have opened the preview link and:

- [ ] checked that this PR resolves the spec provided from trello / this description
- [ ] checked the effected pages at all breakpoints
- [ ] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

File: src/pages/case-study/canon-giving-stories-a-new-form.js [hunk 1]
```diff
@@ -84,7 +80,7 @@ const GradientContent = ({ text, image }) => (
       </CenteredCol>
     </Row>
     <Row>
-      <CenteredCol width={[1, 1, 1, 8 / 12, 7.7 / 12, 7.43 / 12]}>
```
Comment (SaraVieira) [inline src/pages/case-study/canon-giving-stories-a-new-form.js]: wtf was this??

was this me?

File: src/pages/case-study/canon-giving-stories-a-new-form.js [hunk 2]
```diff
@@ -176,18 +172,18 @@ const IndexPage = ({
           </Grid>
         </Padding>
 
-        <NoMobile tablet>
-          <GradientGrid>
+        <NoMobile tablet as={Grid}>
```
Comment (SaraVieira) [inline src/pages/case-study/canon-giving-stories-a-new-form.js]: Nice!

Comment (antoniocapelo) [reply src/pages/case-study/canon-giving-stories-a-new-form.js]: 😊

File: src/pages/case-study/canon-giving-stories-a-new-form.js (hunk 1 above)
Comment (SaraVieira) [reply src/pages/case-study/canon-giving-stories-a-new-form.js]: 7.7 / 12, 7.43 / 12 === 😍

Comment (antoniocapelo) [reply src/pages/case-study/canon-giving-stories-a-new-form.js]: I thought the same 🤷‍♂️

**Pull request merged**